terraform/azure/outputs.tf

```diff
@@ -5,6 +5,11 @@
 # - Public node name
 
 # iSCSI server
+output "hana_os_major_verion" {
+  description = "The major version of the HANA OS"
+  value       = local.hana_major_version
+}
+
 
```

Comment:
If the os_image doesn't match the conventional pattern ("SUSE:sles-sap-15-sp3:gen2:latest"), the major version can be specified with these x_os_major_version.  This is used to set the ansible_python_path